src/lib/Component/ScriptComponent.php

```diff
@@ -55,13 +55,19 @@ public static function getType(): string
      */
     public function render(array $parameters = []): string
     {
-        return $this->twig->render('@ibexadesign/script.html.twig', [
-            'src' => $this->src,
-            'type' => $this->type,
-            'async' => $this->async,
-            'defer' => $this->defer,
-            'crossorigin' => $this->crossorigin,
-            'integrity' => $this->integrity,
-        ] + $parameters);
+        return $this->twig->render(
+            '@IbexaTwigComponents/themes/standard/twig_components/script.html.twig',
```

Comment:
Why changed from `@ibexadesign`? are there some issues or just there is no need to customize that part in anyway?